### ISSUE TITLE
Letter to St. Louis Mayor Lyda Krewson and the Board of Alderman

### ISSUE THREAD

**Issue by jmhstl** [defund12/defund12.org#778]
**To:** krewsonl@stlouis-mo.gov; reedl@stlouis-mo.gov; tyuss@stlouis-mo.gov; middlebrookl@stlouis-mo.gov; bosleyb@stlouis-mo.gov; hubbardt@stlouis-mo.gov; ingrassiac@stlouis-mo.gov; coatarj@stlouis-mo.gov; ricea@stlouis-mo.gov; guentherd@stlouis-mo.gov; vollmerj@stlouis-mo.gov; martins@stlouis-mo.gov; murphyb@stlouis-mo.gov; howardc@stlouis-mo.gov; greenm@stlouis-mo.gov; oldenburgt@stlouis-mo.gov; roddyj@stlouis-mo.gov; toddj@stlouis-mo.gov; davisma@stlouis-mo.gov; spencerc@stlouis-mo.gov; collinsmuhammadj@stlouis-mo.gov; boydj@stlouis-mo.gov; vacarroj@stlouis-mo.gov; narayanb@stlouis-mo.gov; cohns@stlouis-mo.gov; clark-hubbards@stlouis-mo.gov; boydp@stlouis-mo.gov; navarroh@stlouis-mo.gov

**Subject:** [*** INSERT UNIQUE SUBJECT LINE***]

Dear Mayor Krewson and Members of the Board of Alderman,

My name is [YOUR NAME], and I am a resident of [NEIGHBORHOOD/CITY]. Since late May, our nation has been gripped by protests calling for rapid and meaningful reevaluation of the role of policing in our communities and an end to systemic racism and anti-Blackness.

St. Louis is one of the most segregated cities in the United States. And the City’s and region’s history of policy brutality in recent years—from the shooting of Michael Brown and nation-galvanizing outcry thereafter, to the shooting of Anthony Lamar Smith, acquittal of Jason Stockley, and protests that followed—is a stain on our community that must wiped out.

Amidst current cries for change, it has come to my attention that the City’s Proposed 2021 Fiscal Year Operating Plan allocates nearly $278 million of the $495 million general fund budget to public safety, including _nearly $141 million to police services alone_. In other words, _over a quarter of the City’s general fund budget is allocated directly to police services_. By contrast, the same Proposed Budget allocates only $12.7 million to health and hospitals, and only $1.3 million to human services, including social service programs for the elderly, homeless, veterans, people with disabilities, children, and families in need. 

This Proposed Budget is completely blind to how much the St. Louis community has come to over-rely on police services. We turn to the police in situations where years of experience and common sense tell us that their involvement is unnecessary and can make things worse. We ask police to take accident reports, respond to people who have overdosed, roust homeless people from corners and doorsteps, resolve verbal squabbles between family members and strangers alike, and arrest children for behavior that once would have been handled as a school disciplinary issue. Police themselves often complain about having to “do too much,” including handling social problems for which they are ill-equipped. Some have been vocal about the need to decriminalize social problems and take police out of the equation. It is clear that we must re-imagine the role the police play in public safety. 

With this in mind, now is the time to shrink the scope of police responsibilities and shift most of what government does to keep us safe to entities that are better equipped to meet those needs. It means investing more in healthcare (including mental health) and housing, and expanding the use of community mediation and violence interruption programs. It means reducing our reliance on policing to secure our public safety.

I understand that the Proposed Budget will be reviewed and considered by the full Board of Alderman during June 2020. I demand that an emergency public meeting be called before the 2021 fiscal year goes into effect, in order for St. Louisans’ concerns about this budget to be heard. And I call on the Board of Alderman to take the tens of millions of dollars currently allocated to police services, and to responsibly reallocate those funds to underfunded resources that more directly benefit the lives of St. Louisans, including COVID19 relief, healthcare, homelessness, affordable housing, substance abuse treatment, and community centers and other community-led organizations and projects.

We can and must do better. I hope I can count on you to consider an alternative budget that reduces our over-reliance on police services and puts the focus on services that truly benefit public safety our community—particularly in our under-serviced Black and Brown community.

Sincerely,

[YOUR NAME]
[YOUR ADDRESS]
[YOUR EMAIL]
[YOUR PHONE NUMBER]

**Comment by ctneal91** [comment]
seems related to issue #519

**Comment by jmhstl** [comment]
Yes, I included the email addresses for all sitting members of the Board of Alderman.  Thanks.

husgen@gmail.com
718.813.2623

**Comment by paulagodoy** [comment]
vacarroj@stlouis-mo.gov and howardc@stlouis-mo.gov gave mail delivery failures.

**Comment by jmhstl** [comment]
I will try to confirm tomorrow what the correct addresses are. Thanks.

**Comment by jmhstl** [comment]
I was able to update the email address for Alderman Joseph Vaccaro. That had been misspelled, with two r's instead of two c's. The correct address is vaccaroj@stlouis-mo.gov. I was not able to find a correct address for Alderwoman Carol Howard. I just emailed her office through the following portal, however, to request and office address, and hopefully I will hear back tomorrow: https://www.stlouis-mo.gov/government/departments/aldermen/profiles/Carol-Howard.cfm. Thanks.

**Comment by jmhstl** [comment]
And now I have corrected the email address for Alderwoman Carol Howard, because she responded directly to my email to her office. It is howardca@stlouis-mo.gov. The emails for both Alderman Joseph Vaccaro and Alderwoman Carol Howard have been updated in the body of the letter, above, so none of the emails should give delivery failures. Thanks much!
